docs/source/conf.py

```diff
@@ -135,6 +135,10 @@ def setup(app):
 # The full version, including alpha/beta/rc tags.
 release = str(semver_version)
 
+# nitpicky was first enabled in version 0.9
+if semver_version < (0, 9):
+    nitpicky = False
```

Comment:
```suggestion
if semver_version < (0, 9):
    nitpicky = False
else:
    nitpicky = True
    nitpick_ignore = [
        ...
```

Could you move the `nitpick_ignore` definition down here? If someone in the future needs to debug this, it will be nice to have it in one place.